src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,69 +1,81 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import IconQuestion from "@elements/Icon/Generated/IconQuestion";
+import React, { cloneElement, FC, ReactElement, ReactNode } from "react";
+import { Button, ButtonStyle, ButtonSize } from "@elements/Button/Button";
 import { IconSize } from "@elements/Icon/IconSize";
-import { useFocusVisible } from "@react-aria/interactions";
-import { useTooltip, useTooltipTrigger } from "@react-aria/tooltip";
-import { useTooltipTriggerState } from "@react-stately/tooltip";
-import { FOCUS_STYLE } from "@utilities/focusStyle";
-import { merge } from "@utilities/merge";
-import React, { FC, ReactNode, useRef, useState } from "react";
-import { usePopper } from "react-popper";
+import { BrightHeader, BrightHeaderVariants } from "./partials/BrightHeader";
+
+export type TooltipButton = {
+    label: string;
+    action: () => void;
+};
 
 export type TooltipProps = {
     tooltip: ReactNode;
+    tooltipIcon?: ReactElement;
+    heading?: ReactNode;
+    headingIcon?: ReactElement;
+    linkUrl?: string;
+    linkLabel?: string;
+    brightHeader?: BrightHeaderVariants;
+    buttons?: [TooltipButton, TooltipButton];
```

Comment:
I'd suggest typing it as follows and only displaying the first two buttons as you've already done, wdyt?

```suggestion
    buttons?: TooltipButton[];
```

Reply:
The current one has the advantage to force the prop to have 2 element of `TooltipButton` type or `undefined`

Reply:
Is it possible to only define one button which is then primary? At the moment this possibility is not covered.

Reply:
The design don't show anything, maybe you can ask Simone @CelienB

Reply:
There are always two buttons on the mockups. I'd have to check with the designer first.